packages/fast-components-react-msft/src/action-toggle/action-toggle.props.ts

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import { Omit } from "utility-types";
+import {
+    ActionToggleClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    ButtonAppearance,
+    ButtonHandledProps,
+    ButtonManagedClasses,
+    ButtonUnhandledProps,
+} from "../button/button.props";
+
+export enum ActionToggleAppearance {
```

Comment:
I don't believe appearance would be exposed in this scenario. I'm struggling to understand what purpose it would serve considering these appearances change based on state.

This component has two states; selected/toggled and unselected/untoggled. So, we either need to provide selected and unselected styles for each appearance, which would be completely new designs - or we need to base this off the current design in the task.

Reply:
Changed to be hard-coded "primary" for selected state and "lightweight" for unselected.